### ISSUE TITLE
Relocate "ignore all new versions" to versions section

### ISSUE THREAD

**Issue by Donkey-Doug** [NeoApplications/Neo-Store#333]
I think having it in a similar location makes more sense. Now the setting is all the way up top and the releases all the way at the bottom.

**Comment by machiav3lli** [comment]
Nah, the update button is located at the top which is the main object of this setting not the already existing releases.
